tests/test_message_variables.py

```diff
@@ -138,8 +138,8 @@ def test_populate_next_map(monkeypatch, map_name, expected):
     logger.error(f"{parse_layer(map_name)=}")
     monkeypatch.setattr(
         rcon.message_variables.Rcon,
-        "next_map",
-        parse_layer(map_name),
+        "get_next_map",
```

Comment:
I do not understand that change. If I understand correctly, shouldn't test that the variables in a message? The variable for the next map is `next_map`, to my knowledge.

Reply:
yes, it is, however, this is a mocking for the underlying function that is used to _fill_ the `next_map` variable. And this changed at some time from `next_map` to `get_next_map` in the `Rcon` class without being changed here :)